optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/persistence/DistanceRepositoryImplTest.java

```diff
@@ -75,4 +75,16 @@ void should_return_negative_number_when_distance_not_found() {
                 // which turns -1 into -0, so this test wouldn't kill that mutation without the following:
                 .isNotZero();
     }
+
+    @Test
+    void should_delete_distance_by_location_id() {
```

Comment:
`asFarAsIKnowWeUseCamelCaseEvenInTestMethodNames`. Although I see that the damage has already been done long before, and pretty consistently.

Reply:
`theDamageHasBeenDoneALongTimeAgoInAGalaxyFarFarAway`

`im_only_trying_not_to_repeat_the_same_mistake_over_and_over`

On the occasion of starting a project from scratch I took the liberty to try a different approach. I applied it consistently and it has proven to work very well, otherwise I would have reverted it as I have done with other "experiments" that didn't work out very well.

`snake_case` is easier to read and by that it encourages writing a greater amount of small, focused tests that have a clear purpose defined by the method name (which helps to keep the test scope limited). When a test fails it's faster to realize what functionality is broken.

Reply:
We seem to have a difference of opinion. I don't understand how having *longer* test names which look *different* than any other names in the code makes anything better.

Still, if test granularity and understanding of results is what you're after, I suggest you explore test nesting, available in JUnit 5.

Reply:
Hmm, nested tests are more about sharing different setup conditions with different groups of test methods. It solves a different problem but yes, it looks useful. I'll try to use it as soon as I come across a test that would benefit from this approach (#344).

BUT, having seen the JUnit 5 docs chapter about nested test classes, I noticed there is another tool that solves exactly the problem `i_tried_to_solve_using_snake_case` test methods names, and that's `@DisplayName("should do something")` (#345).